cg/cli/workflow/jasen/__init__.py

```diff
@@ -0,0 +1 @@
+"""Init for module"""
```

Comment:
I think this docstring is not necessary

Reply:
Good point, although I have a sense that empty files can sometimes cause some troubles. Should we have a standard for this? :)